code/game/objects/structures/customplaque.dm

```diff
@@ -0,0 +1,123 @@
+			user.visible_message("<span class='notice'>[user] unfastens [src].</span>", \
+								 "<span class='notice'>You unfasten [src].</span>")
+			var/obj/item/customplaque/CP = new (get_turf(user))
+			CP.name = name //Copy over the plaque structure variables to the plaque item we're creating when we unwrench it.
+			CP.desc = desc
+			CP.unengraved = unengraved
+			CP.setDir(dir)
+			qdel(src) //The plaque structure on the wall goes poof and only the plaque item from unwrenching remains.
+		return
+	else if(istype(I, /obj/item/pen/fountain) && unengraved)
+		var/namechoice = input(user, "Title this plaque. (e.g. 'Best HoP Award', 'Great Ashwalker War Memorial')", "Plaque Customization")
+		if(!namechoice)
+			return
+		var/descriptionchoice = input(user, "Engrave this plaque's text.", "Plaque Customization")
+		if(!descriptionchoice)
+			return
+		user.visible_message("<span class='notice'>[user] begins engraving [src].</span>", \
+							 "<span class='notice'>You begin engraving [src].</span>")
+		if(do_after(user, 40, target = src)) //This spits out a visible message that somebody is engraving a plaque, then has a delay.
+			if(!Adjacent(user)) //Make sure user is adjacent still
+				to_chat(user, "<span class='warning'>You have to stand next to the plaque to engrave it!</span>")
+				return
+			name = "\improper [namechoice]" //We want improper here so examine doesn't get weird if somebody capitalizes the plaque title.
+			desc = "The plaque reads: '[descriptionchoice]'"
+			unengraved = FALSE //The plaque now has a name, description, and can't be altered again.
+			user.visible_message("<span class='notice'>[user] engraves [src].</span>", \
+								 "<span class='notice'>You engrave [src].</span>")
+	else if(istype(I, /obj/item/pen) && unengraved)
+		to_chat(user, "<span class='warning'>Your pen isn't fancy enough to engrave this! Find a fountain pen.</span>") //Go steal the Curator's.
+	else
+		return ..()
+
+/obj/item/customplaque/attackby(obj/item/I, mob/user, params) //Same as part of the above, except for the item in hand instead of the structure.
+	if(istype(I, /obj/item/pen/fountain) && unengraved)
+		var/namechoice = input(user, "Title this plaque. (e.g. 'Best HoP Award', 'Great Ashwalker War Memorial')", "Plaque Customization")
+		if(!namechoice)
+			return
+		var/descriptionchoice = input(user, "Engrave this plaque's text.", "Plaque Customization")
+		if(!descriptionchoice)
+			return
+		user.visible_message("<span class='notice'>[user] begins engraving [src].</span>", \
+							 "<span class='notice'>You begin engraving [src].</span>")
+		if(do_after(user, 40, target = src))
+			name = "\improper [namechoice]"
+			desc = "The plaque reads: '[descriptionchoice]'"
+			unengraved = FALSE
+			user.visible_message("<span class='notice'>[user] engraves [src].</span>", \
+								 "<span class='notice'>You engrave [src].</span>")
+	else if(istype(I, /obj/item/pen) && unengraved)
```

Comment:
Same here, do `if-return-if` instead of `if-else`
Also end this with `return ..()`

Reply:
Done.